lib/dotcom_web/views/helpers.ex

```diff
@@ -115,13 +115,26 @@ defmodule DotcomWeb.ViewHelpers do
   def line_icon(%Route{external_agency_name: "Massport", name: name}, _)
       when is_binary(name) do
     route_number = String.slice(name, 0..1)
-    svg("icon-massport-#{route_number}.svg")
+
+    case svg("icon-massport-#{route_number}.svg") do
+      {:error, {:unknown_svg, _}} ->
+        svg("icon-mode-bus-default.svg")
+
+      icon ->
+        icon
```

Comment:
We could remove both of these case statements by just making the `svg/1` function return the bus mode default. Otherwise, you get a strange situation where calling it with an unknown string returns an error and then calls itself.